pkg/strategy/grid2/strategy.go

```diff
@@ -854,9 +873,30 @@ func (s *Strategy) openGrid(ctx context.Context, session *bbgo.ExchangeSession)
 
 	s.logger.Infof("ALL GRID ORDERS SUBMITTED")
 	s.EmitGridReady()
+
+	s.updateOpenOrderPricesMetrics(createdOrders)
 	return nil
 }
 
+func (s *Strategy) updateOpenOrderPricesMetrics(orders []types.Order) {
```

Comment:
Need to delete existing values of the same metric and add new values from scratch.

Reply:
fixed, added Reset call